src/db/migrations/20240408161125_add_monthly_report_column.js

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function up (knex) {
+  return knex.schema.table('subscribers', table => {
+    table.boolean('monthly_monitor_report').defaultTo(false)
```

Comment:
How is this different from `monthly_email_optout` column?

Reply:
The `monthly_email_optout` column was used for a different email campaign. It's used by a script via `getSubscribersWithUnresolvedBreaches`. 

What we building here is a completely different feature, which should not inherit the values that exist in that column

Reply:
We *can* delete that column along with `getSubscribersWithUnresolvedBreaches` and the script `send-email-to-unresolved-breach-subscribers.js` but that should be a separate issue

Reply:
Well I ask because I think @Vinnl is using the old column in https://github.com/mozilla/blurts-server/pull/4400 :)

Reply:
Ah yes, @Vinnl I think if that's the case, we should probably clear out the column first (see my point above about existing values not representing this selection)

Also I quite dislike the idea of a column with names like `opt_out`, which forces us to reverse the boolean :(.. this probably also applies to the existing associated timestamp column

Reply:
It's probably a good idea to delete that column and the script for this exact reason :) But I'm happy to update my code to use a different column with a different value, just let me know which.

Reply:
@Vinnl @rhelmer updated with the removal of the two old columns, https://github.com/mozilla/blurts-server/pull/4400#discussion_r1558276218

Reply:
I have no insight in running time of the migration, so I'll leave the full review to Rob, but two remarks:

- Does Tony actually want this to default to `false`? Or should we set it to `false` for existing or non-Plus subscribers, but default to `true` for new ones?
- Don't forget to update the type definitions in `knex-tables.d.ts` :) Ideally in the same commit as the one adding the migration script, so it's easy to find the relevant migration by running `git blame`.

Reply:
lgtm overall @mansaj but see these comments from @Vinnl. If the default of `false` is find, I think all we need is the type def updates (would be nice if the build caught this somehow).

Reply:
@rhelmer just fyi, we actually want it to default to `true` now after talking to prod. 
updated the type def and added some comments for functions that need to be deleted when we tackle tech debt